### DISCUSSION TITLE
MaxRows on Driver Standings

### DISCUSSION THREAD

**Discussion by buteo72** [marcokreeft87/formulaone-card#16] [Ideas]
Hello, 
First, what an amazing job! It really looks very good. Thanks for all your effort.

I'd like to ask if it is possible to add an extra setting for defining a "MaxRows" setting in the formula_one_sensor_drivers and formula_one_sensor_constructors.

I'd like to 'compress' the list, because it is shown on my Magic Mirror and there I can use all the space there is ;-)

Thank your very much in advance.

Existing:
![HA-MagicMirror](https://user-images.githubusercontent.com/58885863/198711749-cb352117-08da-4a22-ba9d-daf26381cdee.png)

Wished situation

![HA-MagicMirror Compressed](https://user-images.githubusercontent.com/58885863/198712023-7c11e180-4965-4a9a-95b5-469837fe9330.png)

**Comment by marcokreeft87** [comment]
Sorry but that such a specific feature that its not worth the effort to build in. You could try to make the card height less with styles or card mod?

**Comment by buteo72** [comment]
OK, I'll try it in another way. Thanks for your resonce

**Reply by marcokreeft87** [reply]
There is another person asking for this. So its going on the todo list :)

**Comment by buteo72** [comment]
That's nice to hear. Solved it with the markup card and the API sensor in the past.Would be very nice, love to test it if you want.Keep up the good work!

**Comment by marcokreeft87** [comment]
@buteo72 check out https://github.com/marcokreeft87/formulaone-card/issues/123 its in v0.8.1

**Reply by buteo72** [reply]
@marcokreeft87 , thanks for the note and the work... Helps me a lot!
